generator/src/main/java/com/scottlogic/deg/generator/CommandLine/GenerateCommandLine.java

```diff
@@ -129,7 +113,6 @@ public File getProfileFile() {
         return this.profileFile;
     }
 
-    @Override
```

Comment:
Is there a reason the `@Override` annotations have been removed?

Reply:
Forgot to put these back in after refactoring CommandLine / ConfigSource object hierarchies.